src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,23 @@
+@mixin on-phone {
+  @media (min-width: 488px) {
+    .container {
+      --colum-count: 2;
```

Comment:
The variable name `--colum-count` seems to be a typo. It should be `--column-count` to reflect the correct spelling and to follow the code style rules.